### PR TITLE
release 2.5.0

### PR DESCRIPTION
**Pull request by wenchajun**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note
release 2.5.0
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
+## 2.5.0 / 2023-09-13
+### Features
+- Feat: add support to run Fluentd as DaemonSet (#839)
+- Feat: add support for prometheus expoter output plugin for fluentbit (#840)
+- Feat: adding forward input plugin configuration (#843)
+- Feat: support fluentd grok parser plugin (#861) 
+- Feat: add gelf output plugin to fluentbit (#882) 
+- Feat: add fluentbit opentelemetry plugin (#890) 
+- Feat: added serviceAccountAnnotations to FluentBit resource (#898)
```
Comment (benjaminhuo) [inline CHANGELOG.md:9]:
```suggestion
- Feat: Add serviceAccount Annotations to FluentBit resource (#898) 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
+## 2.5.0 / 2023-09-13
+### Features
+- Feat: add support to run Fluentd as DaemonSet (#839)
+- Feat: add support for prometheus expoter output plugin for fluentbit (#840)
+- Feat: adding forward input plugin configuration (#843)
+- Feat: support fluentd grok parser plugin (#861) 
+- Feat: add gelf output plugin to fluentbit (#882) 
+- Feat: add fluentbit opentelemetry plugin (#890) 
+- Feat: added serviceAccountAnnotations to FluentBit resource (#898) 
+- Feat: added Helm priorityClassName mapping to fluentd (#902)
```
Comment (benjaminhuo) [inline CHANGELOG.md:10]:
```suggestion
- Feat: Add Helm priorityClassName mapping to Fluentd (#902) 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+- Feat: add fluentbit http plugin (#904) 
+- Feat: add fluentbit mqtt plugin (#911) 
+- Feat: add fluentbit collectd plugin (#914) 
+
+### ENHANCEMENT
+- Remove Duplicate Cluster parsers in Fluent-bit config. (#853) 
+- Add option to configure hostNetwork daemonset propertie (#863)
+- Helm chart generation of ClusterOutput for loki (#865) (#906)
+- Bump fluentbit to 2.1.8. (#867) 
+- Added SuppressTypeName option to helm, as newer ES needs it for bulk operation (#869)
```
Comment (benjaminhuo) [inline CHANGELOG.md:20]:
```suggestion
- Add SuppressTypeName option to helm, as newer ES needs it for bulk operation (#869) 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+- Feat: add fluentbit collectd plugin (#914) 
+
+### ENHANCEMENT
+- Remove Duplicate Cluster parsers in Fluent-bit config. (#853) 
+- Add option to configure hostNetwork daemonset propertie (#863)
+- Helm chart generation of ClusterOutput for loki (#865) (#906)
+- Bump fluentbit to 2.1.8. (#867) 
+- Added SuppressTypeName option to helm, as newer ES needs it for bulk operation (#869) 
+- Adjusting the parameters of fluentbit (#880) @wenchajun
+- Adding an extras section to the chart allowing new and less used features of the CRD to be used from the chart (#889)
```
Comment (benjaminhuo) [inline CHANGELOG.md:22]:
```suggestion
- Add an extras section to the chart allowing new and less used features of the CRD to be used from the chart (#889)
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

**Comment by wenchajun** [comment]
done

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**